### PR TITLE
amp build command

### PR DESCRIPTION
**Pull request by generalhenry**
https://github.com/appcelerator/amp/issues/42

to test run the commands from the issue

### CODE REVIEW

**Comment by generalhenry** [comment]
note: test is failing CI due to kafka issue, I'm looking into it.

**Comment by subfuzion** [comment]
@generalhenry Go ahead and clean up history. Should be one or two commits (at most).

File: api/client/build.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	return client.CreateProject(ctx, &request)
+}
+
+// RemoveProject removes a project through the amplifier proxy to the build service
+func (a *AMP) RemoveProject(repo string) (*build.Project, error) {
+	a.Connect()
+	defer a.Disconnect()
+	client := build.NewAmpBuildClient(a.Conn)
+	ctx := context.Background()
+	split := strings.Split(repo, "/")
```
Comment (subfuzion) [inline api/client/build.go]: Being a bit fussy here, but I would prefer to see all of the places where you are doing this standardized in github helper that splits strings for orgs, repos, builds, etc, and returns what you need or the error.

Comment (generalhenry) [reply api/client/build.go]: Ok, to start I'll refactor into more funcs.  I might also add some regex.

**Comment by subfuzion** [comment]
@generalhenry Excellent, nice work, tests all pass. Only the one outstanding comment above, everything else LGTM.

**Comment by subfuzion** [comment]
One issue: running `amp build logs` without an argument results in a panic.

```
$ amp build logs
amp (cli version: 1.0.0, build: 8445ea98)
panic: runtime error: index out of range
```

User input errors (missing args, invalid options, etc) should not result in a panic. Need to inspect user input and if not valid, provide an error message and exit (eg, `log.Fatalf`).

**Comment by subfuzion** [comment]
Unexpected result:

```
$ amp build listbuilds -lq appcelerator/amp
amp (cli version: 1.0.0, build: 8445ea98)
appcelerator/amp/c4015d02fbc60583a4cd82187eb99d3aac3b36e4

$ amp (build-cli)$ amp build logs c4015d02fbc60583a4cd82187eb99d3aac3b36e4
amp (cli version: 1.0.0, build: 8445ea98)
invalid build ["c4015d02fbc60583a4cd82187eb99d3aac3b36e4"]
```

Why invalid build? The build was provided by the previous query.

**Comment by generalhenry** [comment]
re: panic  I'm adding input validation for the args

re: invalid build it should be `amp build logs appcelerator/amp/c4015d02fbc60583a4cd82187eb99d3aac3b36e4`

**Comment by subfuzion** [comment]
Nice! LGTM

**Pull request merged**